makelib/helm.mk

```diff
@@ -186,3 +205,10 @@ helm.help:
 
 help-special: helm.help
 
+# ====================================================================================
+# Tools install targets
+
+$(HELM_DOCS):
```

Comment:
nit: what do you think about moving this to [here](https://github.com/upbound/build/blob/85713f83bd601a0d415386f12a2adff27348bd4b/makelib/k8s_tools.mk#L153) so that we can have all tools together?